.github/workflows/verify-generation-config.yaml

```diff
@@ -30,3 +72,40 @@ jobs:
           -e "REPO_BINDING_VOLUMES=${repo_volumes}" \
           gcr.io/cloud-devrel-public-resources/java-library-generation:latest \
           python /src/generate_repo.py generate --generation-config-yaml=/workspace/google-cloud-java/generation_config.yaml --repository-path=/workspace/google-cloud-java
+    - name: generate pull request description
+      id: generate-description
+      shell: bash
+      run: |
+        # the pr description (body) will be available in pr-description.txt after
+        # running this command.
+        set -x
+        docker run --rm \
+          ${repo_volumes} \
+          -v /tmp:/tmp \
+          -v /var/run/docker.sock:/var/run/docker.sock \
+          -e "RUNNING_IN_DOCKER=true" \
+          -e "REPO_BINDING_VOLUMES=${repo_volumes}" \
+          gcr.io/cloud-devrel-public-resources/java-library-generation:latest \
+          python /src/generate_pr_description.py generate --generation-config-yaml=/workspace/google-cloud-java/generation_config.yaml --baseline-commit="${baseline_commit}"
+      env:
+        baseline_commit: ${{ env.baseline_commit }}
+    - name: create or update the pull request
+      shell: bash
+      run: |
+        title="chore: generate libraries at $(date)"
```

Comment:
My instinct says "chore:" does not goes to Release Please pull request. We can fix it by "BEGIN_COMMIT_OVERRIDE" if it really does not work. Let's see.

Reply:
I feel the same way. Can we make sure we understand the behavior before merging? e.g. If it is a `chore`, is release please going to include the NESTE_COMMIT in the description? If it is a `feat`, is the title going to be included? The behavior of release-please will affect how we generate the PR descriptions.

Reply:
According to @chingor13, release-please won't skip a commit if it's a chore, it just ignores chore commit messages.